packages/core/src/external/aws/document-signing/document-bulk-signer.ts

```diff
@@ -0,0 +1,10 @@
+export type DocumentBulkSignerRequest = {
+  patientId: string;
+  cxId: string;
+  requestId: string;
+};
+
+export abstract class DocumentBulkSigner {
+  constructor(readonly region: string) {}
+  abstract sign({ patientId, cxId, requestId }: DocumentBulkSignerRequest): void | Promise<void>;
```

Comment:
Why not always a Promise? Can just make the function `async` on all implementations. It keeps this simpler.